MONGODB/F1.4.md

```diff
@@ -0,0 +1,108 @@
+# F1.3: Creating the webpage for the Stitch Application 
+
+Let's generate an HTML page so we can comment on the blog.
+
+## Step 1: Creating the HTML page
+
+Create an HTML file and copy the following onto it:
+
+```html
+<html>
+  <head>
+  </head>
+  <body>
+    <h3>My First Blog</h3>
+    <div id="content">
+      Hi! This is an example of how to use Stitch!
+    </div>
+    <hr>
+    <div id="comments"></div>
+  </body>
+</html>
+```
+
+## Step 2: Include the Stitch SDK
+
+Include the following <script>  in the <head> file. This allows our HTML page to link and communicate with the Stitch application.
+
+```javascript
+<script src="https://s3.amazonaws.com/stitch-sdks/js/bundles/4/stitch.js"></script>
+```
+
+## Step 3: Initialize the Stitch client
+
+Paste this below the SDK in the head tag:
+
+Replace the "<your-app-id>" with your app id, which you can find on the clients page of the Stitch homepage
+
+```javascript
+<script>
+  const client = stitch.Stitch.initializeDefaultAppClient("<your-app-id>");
+  const mongodb = client.getServiceClient(
+    stitch.RemoteMongoClient.factory,
+    "mongodb-atlas"
+  );
+  const db = mongodb.db("blog");
+</script>
+```
+
+## Step 4: Displaying Comments
+
+Add the following functions to the <script> tag:
+
+```javascript
+function displayComments() {
+  db.collection("comments")
+    .find({}, {limit: 1000})
+    .toArray()
+    .then(docs => {
+      const html = docs.map(doc => `<div>${doc.comment}</div>`);
+      document.getElementById("comments").innerHTML = html;
+    });
+}
+```
+
+```javascript
+function displayCommentsOnLoad() {
+  client.auth
+
+    .loginWithCredential(new stitch.AnonymousCredential())
+
+    .then(displayComments)
+    .catch(console.error);
+}
+```
+
+Add this function in the <body> tag:
+
+```javascript
+<body onload="displayCommentsOnLoad()">
+```
+
+## Step 5: Add a form 
+
+Add this function to the <script> tag:
+
+```javascript
```

Comment:
Can you explain what each function is suppose to do? Besides just say "insert this function". Rn all they are doing is just following instruction but remember the end goal is that the student learns

Reply:
This goes for this entire activity